integration_tests/e2e/login.cy.ts

```diff
@@ -20,20 +20,20 @@ context('SignIn', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('User name visible in header', () => {
+  it.skip('User name visible in header', () => {
```

Comment:
Just skipping these tests that relate to HMPPS Auth integration for now - will all be stripped out when moving to GOV.UK One Login...